tests/wp-unit/include/Core/Metas/Modules/DeleteUserMetaModuleTest.php

```diff
@@ -29,18 +32,20 @@ public function setUp() {
 	/**
 	 * Add to test single user meta from admin user role.
 	 */
-	public function test_deleting_single_user_meta_fields_from_admin_user_role() {
+	public function test_that_meta_fields_from_a_single_user_in_admin_role_can_be_delete() {
 		// Create a user with admin role.
 		$user = $this->factory->user->create( array( 'role' => 'administrator' ) );
 
 		add_user_meta( $user, 'time', '10/10/2018' );
 
+		add_user_meta( $user, 'another', 'Another' );
```

Comment:
In `add_user_meta` you are using the meta value `Another`. But in `delete_user_meta` you are using the value `another`.

Store the meta value in a variable and then try to use the variable everywhere.

This is happening in other test cases as well. Fix in all of them.

Reply:
Fixed